lighthouse-core/gather/driver.js

```diff
@@ -150,7 +150,11 @@ class Driver {
         returnByValue: true
       }).then(result => {
         clearTimeout(asyncTimeout);
-        resolve(result.result.value);
```

Comment:
yikes, and `wasThrown` wasn't set and the exceptionDetails has very little information. even more concerning is that I guess because the message is JSONified, if the promise is rejected with an actual error object you lose the message entirely since `JSON.stringify(new Error('message')) === '{}'`